include/coreir/passes/transform/clock_gate.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "coreir.h"
+
+namespace CoreIR {
+
+  namespace Passes {
```

Comment:
nit: indent not needed for nested namespaces

Reply:
Fixed